### ISSUE TITLE
Mapbox not working

### ISSUE THREAD

**Issue by shivapersad** [Yermo/nativescript-mapbox#22]
TNS Version: 2.1.1

When I run "tns livesync ios --emulator --watch" I get the following error messages:

Aug  5 14:51:44 Shivas-Air cngMobileApp[70894]: 1   0x45b82c -[TNSRuntime executeModule:]
Aug  5 14:51:44 Shivas-Air cngMobileApp[70894]: 2   0x8c60b main
Aug  5 14:51:44 Shivas-Air cngMobileApp[70894]: 3   0x3cc5a25 start
Aug  5 14:51:44 Shivas-Air cngMobileApp[70894]: file:///app/tns_modules/nativescript-mapbox/mapbox.js:7:20: JS ERROR ReferenceError: Can't find variable: MGLOfflineStorage
Aug  5 14:51:44 Shivas-Air com.apple.CoreSimulator.SimDevice.20124C04-21CD-486F-9DFE-439512DD31BD.launchd_sim[65148](UIKitApplication:org.nativescript.cngMobileApp[0xc842][70894]): Service exited due to signal: Segmentation fault: 11

I installed nativescript-mapbox using "tns plugin add nativescript-mapbox". Can you assist me as to what could be causing the issue? Thanks.

**Comment by EddyVerbruggen** [comment]
Perhaps you're trying to show a map before the app has started?

Please show me the code reproducing this issue as I can't get it to break..

**Comment by shivapersad** [comment]
Got it to work. I'm using the angular version of Typescript so I confused some of the code and this may have caused the issue. 

Thanks for the reply all the same.

**Comment by ttma1046** [comment]
I had the same issue. How did you fix it?

JS: Angular 2 is running in the development mode. Call enableProdMode() to enable the production mode.
JS: ReferenceError: Can't find variable: MGLOfflineStorage

I followed the steps what they do during last night's NativeScript 2.3.0 webinar by using Angular 2 typescript with NativeScript to build an ios app.

https://www.youtube.com/watch?v=WQktviwAGb0

I generate a token on mapbox.com after I registered an account and created a token with everything ticked.
